politeiawww/api/v1/api.md

```diff
@@ -1222,6 +1224,119 @@ Reply:
 }
 ```
 
+### `Submit comment`
+
+Submit comment on given proposal.
+
+**Route:** `POST /v1/comments/comment`
+
+**Params:**
+
+| Parameter | Type | Description | Required |
+| - | - | - | - |
+| Token | string | Censorship token | Yes |
+| ParentID | uint64 | Parent comment | Yes |
+| Comment | string | Comment | Yes |
+
+**Results:**
+
+| | Type | Description |
+| - | - | - |
+| CommentID | uint64 | Server generated comment ID |
+| ErrorCode | StatusT | Error code |
+
+**Example**
+
+Request:
+
+```json
+{
+  "token":"05821d2e36e84905c492a21b2cc170952f7536c63a6977bf0c1a5718c1b0fc0d",
+  "comment":"I like this prop"
+}
+```
+
+Reply:
+
+```json
+{
+  "commentid":42,
+  "errorcode":1
+}
+```
+
+### `Get comments`
+
+Retrieve all comments for given proposal.  Not that the comments are not
+sorted.
+
+**Route:** `GET /v1/comments/get`
+
+**Params:**
+
+| Parameter | Type | Description | Required |
+| - | - | - | - |
+| Token | string | Censorship token | Yes |
+
+**Results:**
+
+| | Type | Description |
+| - | - | - |
+| Comments | CookedComment | Unsorted array of all comments |
+| ErrorCode | StatusT | Error code |
+
+**CookedComment:**
+
+| | Type | Description |
+| - | - | - |
+| CommentID | uint64 | Unique comment identifier |
+| UserID | uint64 | Unique user identifier |
+| Timestamp | int64 | UNIX time when comment was accepted |
+| Comment | Comment | Comment content |
+
+**Comment:**
+
+| | Type | Description |
+| - | - | - |
+| Token | string | Censorship token |
+| ParentID | uint64 | Parent comment |
+| Comment | string | Comment text |
+
+**Example**
+
+Request:
+
+```json
+{
+  "token":"05821d2e36e84905c492a21b2cc170952f7536c63a6977bf0c1a5718c1b0fc0d"
+}
+```
+
+Reply:
+
+```json
+{
+  "cookedcomments":[{
```

Comment:
Why is this called cookedcomments and not just comments?

Is cooking something clients should care about?

Reply:
Done

Reply:
It still says cookedcomments in the docs